ddtrace/appsec/_ddwaf/__init__.py

```diff
@@ -54,7 +54,7 @@ def __init__(
         runtime: float,
         total_runtime: float,
         timeout: bool,
-        truncation: int,
+        truncation: _observator,
```

Comment:
rename truncation to 'observator'?

Reply:
this is an observator object for truncation event. So it's ok to use the truncation name here.